css/icons.scss

```diff
@@ -4,6 +4,9 @@
 .icon-deck {
 	background-image: url('../img/deck-dark.svg');
 }
+body.dark .icon-deck {
```

Comment:
to me this sounds like a hack.
With the navigation icon there is a `filter: invert` applied.

I see a similar problem now with the dashboard and unified search. I think we should think about a generic solution to this problem.

Reply:
I've removed the whole hardcoded .icon-deck now since it was a fallback for times before we had the icon-back-white mixin anyways.